integration/spark2/src/main/scala/org/apache/spark/sql/parser/CarbonSpark2SqlParser.scala

```diff
@@ -213,6 +217,12 @@ class CarbonSpark2SqlParser extends CarbonDDLSqlParser {
         TableIdentifier(tableName, dbName)
     }
 
+  protected lazy val forTable: Parser[TableIdentifier] =
```

Comment:
please reuse this parser for the following command:
show segment, clean file, carbon CLI

Reply:
Done.